src/components/families/family-form/family-parent-fields/FamilyParentFields.tsx

```diff
@@ -14,91 +14,101 @@ type Props = {
-      onChange={(value) => {
-        const dob = value || null;
-        onChange({
-          ...family,
-          parent: {
-            ...family.parent,
-            date_of_birth: dob,
-          },
-        });
-      }}
-      value={family.parent.date_of_birth || ""}
-    />
-    <Field
-      field={{ ...DefaultFields.HOME_NUMBER, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, home_number: value })}
-      value={family.home_number}
-    />
-    <Field
-      field={{ ...DefaultFields.CELL_NUMBER, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, cell_number: value })}
-      value={family.cell_number}
-    />
-    <Field
-      field={{ ...DefaultFields.WORK_NUMBER, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, work_number: value })}
-      value={family.work_number}
-    />
-    <Field
-      field={{ ...DefaultFields.PREFERRED_NUMBER, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, preferred_number: value })}
-      value={family.preferred_number}
-    />
-    <Field
-      field={{ ...DefaultFields.EMAIL, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, email: value })}
-      value={family.email}
-    />
-    <Field
-      field={{ ...DefaultFields.ADDRESS, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, address: value })}
-      value={family.address}
-    />
-    <Field
-      field={{ ...DefaultFields.PREFERRED_CONTACT, role: StudentRole.PARENT }}
-      onChange={(value) => onChange({ ...family, preferred_comms: value })}
-      value={family.preferred_comms}
-    />
-    {dynamicFields.map((field) => (
+const FamilyParentFields = ({ dynamicFields, family, onChange }: Props) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const fieldProps = {};
+  return (
+    <>
       <Field
-        key={field.id}
-        field={field}
+        field={{ ...DefaultFields.FIRST_NAME, role: StudentRole.PARENT }}
         onChange={(value) =>
+          onChange({
+            ...family,
+            parent: { ...family.parent, first_name: value },
+          })
+        }
+        value={family.parent.first_name}
+      />
+      <Field
+        field={{ ...DefaultFields.LAST_NAME, role: StudentRole.PARENT }}
+        onChange={(value) =>
+          onChange({
+            ...family,
+            parent: { ...family.parent, last_name: value },
+          })
+        }
+        value={family.parent.last_name}
+      />
+      <Field
+        field={{ ...DefaultFields.DATE_OF_BIRTH, role: StudentRole.PARENT }}
+        onChange={(value) => {
+          const dob = value || null;
           onChange({
             ...family,
             parent: {
               ...family.parent,
-              information: {
-                ...family.parent.information,
-                [field.id]: value,
-              },
```

Comment:
wait a minute, I don't remember seeing this in #68. I originally imagined this PR would be branched off the branch for #68, is that not the case?